modules/acquia_cms_article/acquia_cms_article.install

```diff
@@ -19,3 +19,12 @@ function acquia_cms_article_install() {
     'delete any article content',
   ]);
 }
+
+/**
+ * Implements hook_update_N().
```

Comment:
Instead of writing hook_update_N, can we add some details what this update is doing here?

Reply:
Done